services/visual_recognition/v3.js

```diff
@@ -155,6 +108,12 @@ module.exports = function (RED) {
       apikey = sAPIKey || this.credentials.apikey;
       this.status({}); 
 
+      var visualRecognition = watson.visual_recognition({
+          api_key: apikey,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 space characters but found 10.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2713769514/issues/source?bid=3225115&fileBranchId=3344975#l112)